docs/authorization/globus_app/apps.rst

```diff
@@ -0,0 +1,89 @@
+.. _globus_apps:
+
+.. currentmodule:: globus_sdk
+
+GlobusApps
+==========
+
+
+.. warning::
+
+    Currently ``GlobusApp`` only supports scripting-based use cases.
+
+    Hosted applications (i.e. web servers) should see :ref:`globus_authorizers` instead.
+
+
+:class:`GlobusApp` is a high level construct designed to simplify the process of
+of authenticating for Globus service clients.
+
+A ``GlobusApp`` will always require a programmatic entity called a "client". Depending
+on the type of ``GlobusApp`` in use, the client will play different roles in the
+request flow.
+
+*   A :class:`UserApp` drives interaction starting with a browser based login
+    flow. With this type of ``GlobusApp``, newly created resources (collections,
+    flows, etc.) are owned by the human user who has logged in (and thus granted the
+    app authorization tokens). Similarly resource access is evaluated based on that
+    user's permissions.
```

Comment:
```suggestion
*   A :class:`UserApp` drives interaction using a web-browser-based login
    flow. With this type of ``GlobusApp``, all service interactions are performed as the
    Globus-identity of the user who logged in. For instance, newly created resources
    (collections, flows, etc.) may be owned by that user. Similarly, access to resources
    will be evaluated based on that user's permissions.
```

Reply:
This is now in the table with updated wording.